web/versions.go

```diff
@@ -230,6 +242,34 @@ func getVersionAttachment(c echo.Context, filename string) error {
 	return c.Stream(http.StatusOK, contentType, att.Content)
 }
 
+func getVersionAttachment(c echo.Context, filename string) error {
+	virtualSpace, space, err := getVirtualSpace(c)
+	if err != nil {
+		return err
+	}
+
+	slug := c.Param("app")
+	version := c.Param("version")
+	ver, err := registry.FindVersion(space, slug, version)
+	if err != nil {
+		return err
+	}
+
+	var att *registry.Attachment
+	if virtualSpace != nil {
+		if att, err = registry.FindAttachmentFromOverwrite(virtualSpace, slug, filename); err != nil {
+			return err
+		}
+	}
+	if att == nil {
+		if att, err = registry.FindVersionAttachment(space, ver, filename); err != nil {
+			return err
+		}
+	}
```

Comment:
I would prefer:

```go
var att *registry.Attachment
if virtualSpace != nil {
	att, err = registry.FindAttachmentFromOverwrite(virtualSpace, slug, filename)
} else {
	att, err = registry.FindVersionAttachment(space, ver, filename)
}
if err != nil {
	return err
}

Reply:
Not equivalent. If we don't find an overwrite, we have to use try the standard one.

Reply:
Yep, I forgot that FindAttachmentFromOverwrite can `return nil, nil`

Reply:
I fix the code anyway to also return a boolean too